src/query/expression/src/values.rs

```diff
@@ -1972,12 +1972,16 @@ impl ColumnBuilder {
             }
             ScalarRef::Map(col) => ColumnBuilder::Map(Box::new(ArrayColumnBuilder::repeat(col, n))),
             ScalarRef::Bitmap(b) => {
-                let rb =
-                    RoaringTreemap::deserialize_from(*b).expect("failed to deserialize bitmap");
-                let mut buf = vec![];
-                rb.serialize_into(&mut buf)
-                    .expect("failed to serialize bitmap");
-                ColumnBuilder::Bitmap(StringColumnBuilder::repeat(&buf, n))
+                if b.is_empty() {
+                    ColumnBuilder::Bitmap(StringColumnBuilder::repeat(b, n))
+                } else {
+                    let rb =
+                        RoaringTreemap::deserialize_from(*b).expect("failed to deserialize bitmap");
```

Comment:
It seems that deserialize is not necessary, we can just call the repeat function